source/parameters/beambeam.md

```diff
@@ -1,5 +1,18 @@
 (s:beambeam.params)=
-## BeamBeamP: Beambeam Parameters
+## BeamBeamP: BeamBeam Parameters
 
-In Construction...
+The `BeamBeamP` parameter group describes a particle beam element
+from the opposite moving colliding beam. 
+
+The inputs of `BeamBeamP` are:
+```{code} yaml
+BeamBeamP:
+  sigx      # The horizontal beam size of the opposite beam (default: 0 m).
+  sigy      # The vertical beam size of the opposite beam (default: 0 m).
+  xdisp     # The horizontal displacement of the opposite beam (default: 0 m).
+  ydisp     # The vertical displacement of the opposite beam (default: 0 m).
```

Comment:
Displacement and orientation can be handled by the BodyShiftP parameter group and do not need to be duplicated.